Introduction and Conclusion (#73)
SOMAS2020/SOMAS-Final-Report: 00_introduction/index.tex

\chapter{Introduction}

\begin{flushleft}
\begin{quote}

>     "Therein is the tragedy. Each man is locked into a system that compels him to increase his herd without limit – in a world that is limited. Ruin is the destination toward which all men rush, each pursuing his own best interest in a society that believes in the freedom of the commons."

Reviewer: ```suggestion
    ``Therein is the tragedy. Each man is locked into a system that compels him to increase his herd without limit – in a world that is limited. Ruin is the destination toward which all men rush, each pursuing his own best interest in a society that believes in the freedom of the commons.''
```

    \linebreak
    \emph{Ellinor Ostrom}
\end{quote}
\end{flushleft}

> In the current political climate, while shaking a fist angrily at the television screen, one often finds oneself asking the question, 'are human beings even capable of creating institutions for the common good?' As such, it is worth investigating whether electronic multi-agent systems are able to do a better than mere mortals, in the eventual hope of designing stronger, fairer socio-technical systems.

Reviewer 2: ```suggestion
In the current political climate, while shaking a fist angrily at the television screen, one often finds oneself asking the question, 'Are human beings even capable of creating institutions for the common good?'. As such, it is worth investigating whether electronic multi-agent systems are able to do a better than mere mortals, in the eventual hope of designing stronger, fairer socio-technical systems.
```

> In the current political climate, while shaking a fist angrily at the television screen, one often finds oneself asking the question, 'Are human beings even capable of creating institutions for the common good?'. As such, it is worth investigating whether electronic multi-agent systems are able to do a better than mere mortals, in the eventual hope of designing stronger, fairer socio-technical systems.

Reviewer: ```suggestion
In the current political climate, while shaking a fist angrily at the television screen, one often finds oneself asking the question: ``Are human beings even capable of creating institutions for the common good?''. As such, it is worth investigating whether electronic multi-agent systems are able to do a better than mere mortals, in the eventual hope of designing stronger, fairer socio-technical systems.
```

This project explores the ability of independent agents to self-organise in order to solve long and short-term collective risk dilemmas. The collective risk dilemma poses a challenge to a group of agents whereby they must balance their own interests against the interests of the collective. 

> The objective of this project was to design and implement two collective risk dilemmas as well as a system of agents capable of mitigating the risk. In order to do this successfully, the system contains adequate communication and self organisation tools for the agents to make use of.

Reviewer 2: ```suggestion
The objective of this project was to design and implement two collective risk dilemmas as well as a system of agents capable of mitigating the risk. In order to do this successfully, the system contains adequate communication and self-organisation tools for the agents to make use of.
```

> The specific problem posed can be summarised as follows: islands in an archipelago are incentivised to pool their resources in order to collectively survive environmental disasters but must balance this against an interest in their own survival and properity. The method of resource replenishment is deer hunting and fishing. The populations of these resources are limited and neither of these foraging methods have guaranteed returns. 

Reviewer 2: ```suggestion
The specific problem posed can be summarised as follows: Islands in an archipelago are incentivised to pool their resources in order to collectively survive environmental disasters but must balance this against an interest in their own survival and prosperity. The method of resource replenishment is deer hunting and fishing. The populations of these resources are limited and neither of these foraging methods has guaranteed returns. 
```
